src/VirtualDesk.cpp

```diff
@@ -184,5 +184,5 @@ Layout VirtualDesk::generateCurrentMonitorLayout() {
 std::string VirtualDesk::monitorDesc(const CMonitor* monitor) {
     if (!monitor->output)
         return monitor->szName;
-    return monitor->output->description ? monitor->output->description : monitor->szName;
+    return monitor->output->description;
```

Comment:
I think this should actually be:
```suggestion
    return monitor->output->description.empty() ? monitor->szName : monitor->output->description;
```